main/mySpace/myStudents.php

```diff
@@ -2076,7 +2089,13 @@ function ($hookHeader) {
     $row++;
     foreach ($users as $userFollowed) {
         $followedUserId = $userFollowed['user_id'];
-        $url = api_get_path(WEB_CODE_PATH).'tracking/messages.php?from_user='.$student_id.'&to_user='.$followedUserId;
+
+        if ($filter_messages) {
+            $url = api_get_path(WEB_CODE_PATH).'tracking/messages.php?from_user='.$student_id.'&to_user='.$followedUserId.'&date_from='.$coach_access_start_date.'&date_to='.$coach_access_end_date;
```

Comment:
Use the same variables in the URL: 
"start_date" instead of "date_from" and "date_to" to "end_date" if that's the case.

Reply:
Fixed!